search-app/src/cmr/search/site/util.clj

```diff
@@ -0,0 +1,107 @@
+(ns cmr.search.site.util
+  "Search site support functions."
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clojure.java.io :as io]
+   [cmr.common-app.static :as static]
+   [cmr.common.log :refer :all]
+   [cmr.transmit.config :as transmit])
+  (:import (java.net ConnectException)))
+
+(def supported-directory-tags
+  "A utility data structure used for both the tag names themselves as well as a
+  mapping to more human-friendly version of a tag."
+  {"gov.nasa.eosdis" "EOSDIS"})
+
+(defn get-app-url
+  "A utility function for gettin the app's root URL.
```

Comment:
getting typo

Reply:
Added the missing apostrophe:

"gettin'"

Just kidding -- fixed.